tcp/server/config.go

```diff
@@ -13,8 +13,8 @@ import (
 	"github.com/plgd-dev/go-coap/v3/tcp/client"
 )
 
-// A ServerOption sets options such as credentials, codec and keepalive parameters, etc.
-type TCPServerOption interface {
+// A Option sets options such as credentials, codec and keepalive parameters, etc.
+type TCPOption interface {
```

Comment:
What's the difference between `TCPOption` and `Option` from `tcp/server/server.go` ? Both of these have the same single method - `TCPServerApply(*Config)`. Seems like duplicity.

Reply:
This was duplicity - removed